### PR TITLE
document bounds

### PR DESCRIPTION
**Pull request by jw3126**
I think `bounds` is quite useful and should be part of the public API.

### CODE REVIEW

**Comment by jw3126** [comment]
Would it be okay to also export bounds?

**Comment by timholy** [comment]
This seems reasonable to me, with perhaps one caveat. Suppose someday we define a `DelaunyInterpolation` type; in that case `bounds` should describe a pretty complicated region. Would we be signing ourselves up for trouble?

**Comment by jw3126** [comment]
Good point. We could define bounds only in easy cases? Where the bounds can be described as a rectangular region?

File: src/Interpolations.jl [hunk 1]
```diff
@@ -107,6 +107,28 @@ axes(exp::AbstractExtrapolation) = axes(exp.itp)
 
 twotuple(r::AbstractUnitRange) = (first(r), last(r))
 twotuple(x, y) = (x, y)
+
+"""
+    bounds(itp::AbstractInterpolation)
+
+Return the `bounds` of the domain of `itp`. This is best explained by example:
```
Comment (timholy) [inline src/Interpolations.jl]:
```suggestion
Return the `bounds` of the domain of `itp` as a tuple of `(min, max)` pairs for each coordinate. This is best explained by example:
```

**Comment by timholy** [comment]
In theory, adding a new export could be deemed breaking (if it conflicts with an export from another package), but I am inclined to just bump the patch version. I'll hold off for a day or so for feedback before tagging a new release.

**Comment by jw3126** [comment]
Yeah, the name `bounds` probably already exists at some places already.
But one can argue its the consumers responsibility to either avoid "wildcard imports" `using SomePackage` altogether, or deal with surprising new exports from time to time.

**Pull request merged**